### PR TITLE
Update release process.

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/42/
Test PASSed.

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -79,16 +74,22 @@ Our alphas are development versions of the charm and are published to the
...
+    charm2 info --include=id,perm cs:~yellow/juju-gui
+    charm2 info --include=id,perm cs:~yellow/development/juju-gui
 
 Next, to upload the charm, go to the charm source directory and do:
 
     make clean-tests
     # NB: trusty is the default series unless specified in metadata.yaml.
-    charm upload . cs:~yellow/juju-gui
-    charm info --include=id,perm cs:~yellow/juju-gui
+    charm2 upload . cs:~yellow/trusty/juju-gui
```
Comment (jrwren) [inline RELEASE_PROCESS.md]: We know this triggers that juju-gui bug. Maybe we should make a note of that?

**Comment by jrwren** [comment]
:+1: As long as it is ok that we are going to change this all very soon as bugs are fixed and charm2 becomes charm command again.

File: RELEASE_PROCESS.md (hunk 1 above)
Comment (bac) [inline RELEASE_PROCESS.md]: I think this one is ok as we discussed.

**Comment by bac** [comment]
Yeah, the charm2 -> charm thing is a pain.  I'll take the bullet and fix it whenever they commands are unified.

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-charm-merge

**Pull request merged**